src/lib/components/Widget.js

```diff
@@ -56,13 +57,16 @@ class Widget extends React.Component {
   }
 
   render() {
-    const { error, isLoaded, rules } = this.state;
+    const { error, isLoaded, rules, showLogo } = this.state;
     return (
       <div className="rw-container">
-        <div className="rw-title">
-          <a href={this.sswUrl}>
+        
+        <div className="rw-title" style={{marginBottom: '16px'}}>
```

Comment:
Can this be added to the css class "rw-title"?

Reply:
You win this round Jack!